tests/zero_code_change/pt_utils.py

```diff
@@ -58,3 +62,36 @@ def forward(self, x):
         x = F.relu(self.fc2(x))
         x = self.fc3(x)
         return x
+
+
+def helper_torch_train(sim=None, script_mode=False, use_loss_module=False):
+    trainloader, testloader = get_dataloaders()
+    net = Net()
+    criterion = nn.CrossEntropyLoss()
+    optimizer = optim.SGD(net.parameters(), lr=0.001, momentum=0.9)
+
+    if script_mode:
+        hook = smd.Hook(out_dir=sim.out_dir)
+        hook.register_module(net)
+        hook.register_loss(criterion)
+
+    for i, data in enumerate(trainloader, 0):
+        # get the inputs; data is a list of [inputs, labels]
+        inputs, labels = data
+
+        # zero the parameter gradients
+        optimizer.zero_grad()
+
+        # forward + backward + optimize
+        outputs = net(inputs)
+        if use_loss_module:
+            loss = criterion(outputs, labels)
+        else:
+            loss = F.cross_entropy(outputs, labels)
+            if script_mode:
+                hook.record_tensor_value(tensor_name="loss", tensor_value=loss)
+        loss.backward()
+        optimizer.step()
+
+        if i == 499:  # print every 2000 mini-batches
+            break
```

Comment:
I'm confused by this `if` block and the corresponding comment - how does breaking on the 500th iteration of the loop correspond to printing every 2000 mini-batches? My guess is that this comment is outdated.

Reply:
Not sure what the intention of the comment is (I just refactored the code). Will remove the comment since I agree it's outdated.

Reply:
Done.